Library/Homebrew/test/support/fixtures/cask/Casks/invalid/invalid-header-format.rb

```diff
@@ -1,9 +1,9 @@
-cask => 'invalid-header-format' do
+cask 'invalid-header-format', :invalid do
```

Comment:
This wasn't an auto-fix, but with `cask =>`, Rubocop was unable to parse the invalid Ruby.